examples/utxo/start_test.go

```diff
@@ -56,6 +56,12 @@ func TestUtxo(t *testing.T) {
 	}))
 	t.Cleanup(func() {
 		_ = ic.Close()
+		for _, chain := range chains {
+			utxoChain, ok := chain.(*utxo.UtxoChain)
+			if ok {
+				utxoChain.Stop()
+			}
+		}
```

Comment:
Not for this PR, but if this always needs to be called it may make sense to move into the ic.Close() func?